pkg/gitreceive/build.go

```diff
@@ -227,7 +227,8 @@ func build(
 	log.Debug("Done")
 
 	log.Debug(
-		"Polling the S3 server every %s for %s for the resultant slug at %s",
+		"Polling the %s every %s for %s for the resultant slug at %s",
```

Comment:
This will go any way https://github.com/deis/builder/pull/292. Once 292 is merged I'll rebase this and merge